janitor/functions.py

```diff
@@ -2316,6 +2322,8 @@ def row_to_names(
         Defaults to False.
     :param remove_rows_above: Whether the rows above the selected row should
         be removed from the DataFrame. Defaults to False.
+    :param reset_index : Whether the index should be reset on the returning
+        DataFrame. Defaults to True.
```

Comment:
```suggestion
    :param reset_index: Whether the index should be reset on the returning
        DataFrame. Defaults to True.
```

For sphinx-style documentation to work, the colon must be adjacent to the parameter name and not have a space in between them.

Reply:
Thanks I will keep this in mind for next time